src/react/databrowser/objects/__tests__/ObjectList.test.jsx

```diff
@@ -0,0 +1,91 @@
+import * as s3object from '../../../actions/s3object';
+import { BUCKET_NAME, S3_OBJECT, SECOND_S3_OBJECT } from '../../../actions/__tests__/utils/testUtil';
+import { List } from 'immutable';
+import ObjectList from '../ObjectList';
+import React from 'react';
+import { reduxMount } from '../../../utils/test';
+
+describe('ObjectList', () => {
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render ObjectList with no objects', () => {
+        const { component } = reduxMount(<ObjectList objects={List()} bucketName={BUCKET_NAME} prefixWithSlash=''
+            toggled={List()}/>,
+        );
+
+        expect(component.find('Row')).toHaveLength(0);
+    });
+
+    it('should render ObjectList with objects', () => {
+        const { component } = reduxMount(<ObjectList objects={List([S3_OBJECT])} bucketName={BUCKET_NAME}
+            prefixWithSlash=''
```

Comment:
NIT:
```suggestion
            prefixWithSlash='' toggled={List()}/>,
```

(to do for the following tests as well)

Reply:
done